VideoLocker/src/main/java/org/edx/mobile/http/OauthRefreshTokenAuthenticator.java

```diff
@@ -53,23 +57,40 @@ public OauthRefreshTokenAuthenticator(Context context) {
     public Request authenticate(Route route, final Response response) throws IOException {
         logger.warn(response.toString());
 
-        if (!isTokenExpired(response.peekBody(HttpStatus.OK).string())) {
-            return null;
-        }
-
         final AuthResponse currentAuth = loginPrefs.getCurrentAuth();
         if (null == currentAuth || null == currentAuth.refresh_token) {
             return null;
         }
-        final AuthResponse refreshedAuth;
-        try {
-            refreshedAuth = refreshAccessToken(currentAuth);
-        } catch (HttpResponseStatusException e) {
-            return null;
+        String response_body = response.peekBody(200).string();
+
+        String error_code = getErrorCode(response_body);
```

Comment:
Should we make these variables camel case? And the `response_body` variable seems only to be used here so it could be inlined, but that's up to you. It would also be nice to have the variables declared as `final`.